src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheck.java

```diff
@@ -224,10 +224,10 @@ public void visitToken(DetailAST ast) {
         final int colNo = ast.getColumnNo();
         final int lineNo = ast.getLineNo();
         final int[] currentLine = getLineCodePoints(lineNo - 1);
-        final int[] substringAfterToken = CodePointUtil.trim(
+        final int[] substringAfterToken = CodePointUtil.stripTrailing(
                 Arrays.copyOfRange(currentLine, colNo + text.length(), currentLine.length)
         );
-        final int[] substringBeforeToken = CodePointUtil.trim(
+        final int[] substringBeforeToken = CodePointUtil.stripTrailing(
```

Comment:
`trim` was easier to understand, but `stripTrailing` semantics are not obvious here, especially with before/after naming.  Can we just use CodePointUtil#isBlank?

Reply:
Can we just use CodePointUtil#isBlank?

yes, please check I have updated